src/rest/QuerySuggest.ts

```diff
@@ -66,9 +66,40 @@ export interface IQuerySuggestRequest {
    * Specifies the context to use for the request.
    */
   context?: IStringMap<any>;
+  /**
+   * Specifies the second level of origin of the request, typically the identifier of the selected tab from which the request originates.
+   */
+  tab?: string;
```

Comment:
Could you add line breaks in-between the options like the previous ones, it would be easier to read